src/data/automation_i18n.ts

```diff
@@ -374,35 +374,35 @@ export const describeCondition = (
   if (condition.condition === "or") {
     const conditions = ensureArray(condition.conditions);
 
-    let count = "condition";
-
     if (conditions && conditions.length > 0) {
```

Comment:
You reversed this one now?

Reply:
Nice catch, fixed. I intended to reverse that condition but must have missed it.